code-studio/scripts/build-js.js

```diff
@@ -36,6 +36,7 @@ Promise.all([
     filenames: [
       'code-studio.js',
       'levelbuilder.js',
+      'levelbuilder_gamelab.js',
```

Comment:
Note: I'll coordinate with @Bjvanminnen before merging this since it'll conflict with this code-studio => apps merge.